### ISSUE TITLE
Allow order of values in Cesium map legends to be switched

### ISSUE THREAD

**Issue by justinkadi** [NCEAS/metacatui#2574]
**Describe the feature you'd like**
We should be able to switch the order of the values in a map's legend in a data portal. 

**Is your feature request related to a problem? Please describe.**
Currently, we cannot configure the order of the values in a legend's color map to be switched. We can switch the order in the portal config, but it seems like the element is developed so that it is from min <-> max.

**Additional context**
![Screenshot 2024-12-03 at 14 26 12](https://github.com/user-attachments/assets/f2270a24-9712-4506-bb91-fc86424b5c04)

In this screenshot example, Rushiraj tried to switch the order so that positive values were on top and negative values were on the bottom, but it could not be done through the config.

**Comment by robyngit** [comment]
That's true, they are automatically displayed from min to max. @justinkadi could you add a little context as to why it's preferred to show the scale in the opposite direction in this case? Thanks!
